### PR TITLE
Fix issue  #3457 :  Error while uploading to labelbox for annotations

### PR DESCRIPTION
**Pull request by ashish-dahal**
## What changes are proposed in this pull request?

Fix issue  #3457  (minor change)

### Is this a user-facing change that should be mentioned in the release notes?

-   [ ] No. You can skip the rest of this section.
-   [x] Yes. Give a description of this change to be included in the release
        notes for FiftyOne users.

### What areas of FiftyOne does this PR affect?

-   [ ] App: FiftyOne application changes
-   [ ] Build: Build and test infrastructure changes
-   [x] Core: Core `fiftyone` Python library changes
-   [ ] Documentation: FiftyOne documentation changes
-   [ ] Other

### CODE REVIEW

**Comment by brimoor** [comment]
@ashish-dahal I see that you closed the underlying issue https://github.com/voxel51/fiftyone/issues/3457. Does that mean that this PR is no longer needed?

File: fiftyone/core/annotation.py [hunk 1]
```diff
@@ -1,69 +1,2569 @@
 """
-Annotation runs framework.
+Annotation utilities.
```
Comment (brimoor) [inline fiftyone/core/annotation.py:2]: There should not be any changes to `fiftyone.core.annotations`.

File: fiftyone/utils/annotations.py [hunk 2]
```diff
@@ -145,6 +146,12 @@ def annotate(
             one of the supported methods. For existing label fields, if classes
             are not provided by this argument nor ``label_schema``, the
             observed labels on your dataset are used
+        content_type (None): specified the mime type of the provided samples.
```
Comment (brimoor) [inline fiftyone/utils/annotations.py:149]: `Sample.filepath` is always available, so the only reason I could see needing to manually specify `content_type` is if you had a BMP-formatted image stored with a `.jpg` extension for something like that.

**Review by brimoor — CHANGES_REQUESTED** [review]

**Comment by ashish-dahal** [comment]
> @ashish-dahal I see that you closed the underlying issue #3457. Does that mean that this PR is no longer needed?

Yes, I found a simple fix: The Labelbox SDK uses the Python [`mimetypes`](https://docs.python.org/3/library/mimetypes.html#module-mimetypes) to infer image mime types. This, in turn, looks up mime types from `/etc/mime.types` on Linux. On the Databricks cluster I was using, the mime type for BMP images was `image/x-ms-bmp` instead of the expected `image/bmp`, causing Labelbox to throw an incorrect mime error. Solution was to simply update the mime type entry for BMP images in `/etc/mime.types`  to the correct value.

**Comment by brimoor** [comment]
Thanks for closing the loop on this! Will close the PR since a solution has been found 👍

**Pull request closed without merging**